AmazonChimeSDK/AmazonChimeSDK/internal/utils/ImageConversionUtils.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ImageConversionUtils.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import CoreImage
+import UIKit
+
+/// Utils to help converting image byte array to CGImage and CGImage to byte array.
+public class ImageConversionUtils {
+
+    /// A utility function that converts a byte array to a CGImage.
+    ///
+    /// - Parameters:
+    ///   - raw: Pointer to the byte array of the image.
+    ///   - frameWidth: Image width.
+    ///   - frameHeight: Image height.
+    ///   - bytesPerPixel: Total bytes per pixel (color space).
+    ///   - bitsPerComponent: Total bits per component.
+    ///
+    /// - Returns: A CGImage of the input byte array.
+    public class func byteArrayToCGImage(raw: UnsafeMutablePointer<UInt8>,
+                                         frameWidth: Int,
+                                         frameHeight: Int,
+                                         bytesPerPixel: Int,
+                                         bitsPerComponent: Int) -> CGImage? {
+
+        let bytesPerRow = frameWidth * bytesPerPixel
+        let size = frameWidth * frameHeight * bytesPerPixel
+
+        guard let imageCfData = CFDataCreate(nil, raw, size) else {
+            return nil
+        }
+
+        guard let cgProvider = CGDataProvider.init(data: imageCfData) else {
+            return nil
+        }
+
+        guard let cgImage: CGImage = CGImage.init(width: frameWidth,
```

Comment:
nit: you can directly return `CGImage.init()`

Reply:
fixed